modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -470,6 +475,7 @@ GDScriptTokenizer::Token GDScriptTokenizer::annotation() {
 	KEYWORD_GROUP('b')                       \
 	KEYWORD("break", Token::BREAK)           \
 	KEYWORD("breakpoint", Token::BREAKPOINT) \
+	KEYWORD("but", Token::BUT)               \
```

Comment:
I still suggest you replace `but` with `xor`. Since it is more familiar and widespread.

Reply:
Please consider this comment. We prefer not to have aliases unless absolutely necessary, especially in the case of keywords. We need to choose **one** name, and in my opinion `xor` is better than `but`.